tools/report-converter/codechecker_report_converter/coccinelle/output_parser.py

```diff
@@ -23,6 +23,11 @@ def __init__(self, analyzer_result):
         super(CoccinelleParser, self).__init__()
 
         self.analyzer_result = analyzer_result
+        self.checker_name = None
+
+        self.checker_name_re = re.compile(
+            r'^Processing\s*'
+            r'(?P<checker_name>[\S]+\.cocci)\s*$')
```

Comment:
I think we can remove the extension (`.cocci`) from the checker name. So in the example below it would be only `simple` instead of `simple.cocci`. What do you think @jay24rajput, @bulwahn ?

Reply:
Yes, we can do that! I will modify the regex to match only the name of the file. Something like:
```python
r'(?P<checker_name>[\S]+)\.cocci\s*$')
```
If that needs to be done, will also modify ```sample.expected.plist```  accordingly! @bulwahn your call

Reply:
@csordasmarton 's suggestion makes sense; let us do that.